carts/views.py

```diff
@@ -1,3 +1,54 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http.response import JsonResponse
+from django.views         import View
+
+from users.models    import User
+from carts.models    import Cart
+from products.models import Product, ProductOption, Size
+
+class CartView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            #user = request.user    유저에게 토큰 전달한것 인증되면 활성화해서 테스트
+            user = User.objects.get(id=data['user_id'])
+            product = Product.objects.get(id=data['product_id'])    # id와 name 중에 name이 더 직관적이지만 우리가 데이터를 일일이 볼게 아니기도 하고 id가 조금이라도 메모리 공간?을 덜 잡아먹어서 id로 값을 보내는걸까?
+            size = Size.objects.get(id=data['size_id'])
+            quantity = data['quantity']
+            
+            # print('user : ', user)
+            
+            if not Product.objects.filter(id=product.id).exists():
+                return JsonResponse({'message': 'INVALID_PRODUCT'}, status=400)
+            
+            if not Size.objects.filter(id=size.id).exists():
+                return JsonResponse({'message': 'INVALID_SIZE'}, status=400)
+            
+            product_option = ProductOption.objects.filter(product_id=product, size_id=size)
```

Comment:
```suggestion
            if not product.sizes.filter(size_id=size_id).exists():
                return JsonResponse({'message': 'INVALID_PRODUCT'}, status=400)
            
            product_option = product.sizes.get(size_id=size_id)
```
이미 불러온 product와 many to many 관계를 이용하여 product_option을 가져올 수 있을 것 같은데 한 번 shell에서 연습해보면 좋을 것 같아요.